### PR TITLE
IR-471: Adding additional validation

### PR DESCRIPTION
**Pull request by deepsm007**
/cc @flavianmissi

### CODE REVIEW

File: pkg/storage/s3/s3.go [hunk 1]
```diff
@@ -419,7 +419,7 @@ func (d *driver) ConfigEnv() (envs envvar.List, err error) {
 		return
 	}
 
-	if currentFeatureGates.Enabled(util.ChunkSizeMiBFeatureGateName) && d.Config.ChunkSizeMiB > 0 {
+	if currentFeatureGates.Enabled(util.ChunkSizeMiBFeatureGateName) && d.Config.ChunkSizeMiB > 4 && d.Config.ChunkSizeMiB < 5121 {
```
Comment (flavianmissi) [inline pkg/storage/s3/s3.go:422]: Is this range (4 and 5121) defined in the API validation as well? Just curious.

Comment (deepsm007) [reply pkg/storage/s3/s3.go:422]: Yes, minimum=5 and maximum 5120 in kubebuilder validation

**Comment by flavianmissi** [comment]
this is a validation change that matches the one implemented in the API level, so it's not really customer facing.
/label px-approved
/label docs-approved

**Pull request merged**